CONTRIBUTING.md

```diff
@@ -5,26 +5,58 @@ It outlines our workflow and standards for contributing to this project.
 
 ## Table of Contents
 
-- [Pre-commit Hook](#pre-commit-hook)
+- [Pre-commit Hooks](#pre-commit-hooks)
+- [Documentation](#documentation)
+- [Tests](#tests)
 - [GitHub Actions](#github-actions)
 - [Making and Reviewing Changes](#making-and-reviewing-changes)
 - [Notes](#notes)
 
-## Pre-commit Hook
+## Pre-commit Hooks
 
-We use a pre-commit hook to ensure code quality and consistency:
+We use a pre-commit hooks to ensure code quality and consistency:
 
 1. Install pre-commit (see [pre-commit.com](https://pre-commit.com/#install)).
 1. Clone the repository and run `pre-commit install` in the root directory.
-1. The pre-commit hook runs automatically on `git commit`.
+1. The pre-commit hook will now run automatically on `git commit`.
+   - If the hook fails, it will print an error message and abort the commit.
+   - It will also modify the files in-place to fix any issues it can.
+
+## Documentation
+
+We use the [Epytext](https://epydoc.sourceforge.net/epytext.html) markup language for documentation.
+To verify that your documentation is formatted correctly, follow these steps:
+
+1. Download [`get-docs.py`](https://github.com/luxonis/python-api-analyzer-to-json/blob/main/gen-docs.py) script
+1. Run `python3 get-docs.py luxonis_ml` in the root directory.
+   - If the script runs successfully and produces `docs.json` file, your documentation is formatted correctly.
+   - **NOTE:** If the script fails, it might not give the specific error message. In that case, you can run
+     the script for each file individually until you find the one that is causing the error.
+
```

Comment:
Can we suggest something like a VSCode extension that might help you generate correct docs?

Reply:
I wasn't able to find anything. The [autoDocstring](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) extension doesn't support `epytext` (there's an open issue for it, but it's about 3 years old already). It should be supported in PyCharm though, if anyone is using it.

Reply:
@klemen1999 asked me about this and I found [this one](https://marketplace.visualstudio.com/items?itemName=AIC.docify) but haven't tried it